e2e/specs/detail.spec.ts

```diff
@@ -1,11 +1,12 @@
-import { test, expect } from '@playwright/test';
+import test, {expect} from '../framework/pages/pages';
 import { openOnDetailsPageWithFirstHero } from '../framework/utils/setup-teardown-helper';
-import { DetailsPage } from '../framework/pages';
+import { DetailsPage } from '../framework/pages/pages';
 import { PageUrls } from '../framework/constants/page-uri-const';
-import { framework } from '../framework';
+import { Page } from 'playwright';
+import { navigateToDashboardPage, navigateToDetailsPage, navigateToMyHeroesPage } from '../framework/utils/quick-url-navigation';
 
 test.describe.parallel('Tour of Heroes - Detail page e2e tests', async () => {
-    test("Hero details id should equals to url id", async ({ page }) => {
+    test("Hero details id should equals to url id", async ({ page  }) => {
```

Comment:
Extra space added, not needed.
For real project I suggest looking at something like ESLint and including it as a part of your build. It can automatically format files, and also check formatting during PR validation.

Reply:
ok. Will try to add. Now, it's fixed.